webui/react/src/components/WebhookCreateModal.tsx

```diff
@@ -68,7 +70,7 @@ const triggerOptions = [
 ];
 const modeOptions = [
   {
-    label: 'All experiments in Workspace',
+    label: 'All experiments',
     value: V1WebhookMode.WORKSPACE,
```

Comment:
ideally `V1WebhookMode` would be changed so that webhooks without workspaces don't have a "Workspace" mode that has to be overwritten for display.

Reply:
Initially we planed to use "all" and "specific", where "all" stands for all experiment within the cluster/workspace, but we switched to "workspace" because we thought this maybe clearer and more extendable.

Reply:
I understand not changing to "All", but is there a reason not to extend the enum with a "Cluster" or "Global" option?

Reply:
Under our current design, webhooks without workspace are automatically scoped to cluster, if we have a seperate enum "Cluster", then we have to validate for mode "Cluster" the workspace is null, and vise versa.

Reply:
OK personally I think webhooks without a workspace having a "Workspace" mode is a bigger potential problem than the other tradeoffs, but if that was the intentional decision made by the team I won't block on it.